pom.xml

```diff
@@ -25,4 +25,16 @@
             </plugin>
         </plugins>
     </build>
+                
+       <dependencies>
+
+		<!-- Spring framework -->
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring</artifactId>
+			<version>2.5.6</version>
```

Comment:
To jest zbyt stara wersja springa.

Zastosuj najnowszą - 4.3.3.RELEASE

Kolejna sprawa - nie potrzebujesz w tym przypadku całego springa (tzn. wszystkich jego bibliotek) - użyj tylko to co jest niezbędne, nie dodaje się zależności z których aktualnie w kodzie nie korzystasz